asistente_ladm_col/gui/change_detection/dlg_change_detection_settings.py

```diff
@@ -225,19 +238,19 @@ def update_connection_info(self, db_source):
     def accepted(self):
```

Comment:
Please add some documentation to this method

Reply:
Adjustment made, thanks